pages/dkp/kommander/2.1/release-notes/2.1.2/index.md

```diff
@@ -0,0 +1,277 @@
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## Fixes and Improvements
+
+### Flux proxy configuration results in failure to install applications (COPS-7236)
+
+Incorrect Flux proxy configuration was resulting in Kommander applications not pulling and installing.
+
+### Broken FIPS images (COPS-7225)
+
+FIPS images for DKP were overwritten with blank docker images, making deployment fail.
+
+### kube-oidc-proxy error: certificate signed by unknown authority (COPS-7217)
+
+When using a custom domain and TLS certificate issued by Let's Encrypt, the `kube-oidc-proxy` helm chart in the attached cluster did not complete installation and the associated pod has an error.
+
+### Certificate objects updated but not reloading in Kommander pods (COPS-7212)
+
+When looking at the certificate objects from a bundle, none are expired but the pods in the cluster do not reload with current/new certificates, and also display that the certificates were expired.
+
+### Konvoy-image-builder 1.7.0 fails in certain situatios (COPS-7207)
+
+This issue is resolved.
+
+### KIB fails to pull images in airgapped using ct (COPS-7198)
+
+This lead to a problem in air-gapped environments where the images failed to pull in KIB, when using registry mirrors and/or credentials.
+
+### Dashboards redirecting to wrong URL (COPS-7197)
+
+The `traefik-forward-auth-kommander` helm value was overriding ConfigMap with an incorrect authHost value.
+
+### Kommander fails after a clean reinstall (COPS-7193)
+
+There was an unintended name collision between two separate secrets.
+
+### Upgrade konvoy 1.8.4 to 2.1.1 index out of range error (COPS-7183)
+
+When a 1.8 deployment used an existing VPC, the `prepare-to-adopt` command triggered an error.
+
+### Pulling search path from Bastion host breaks DKP cluster (COPS-7181)
+
+When creating an air-gapped cluster on preprovisioned machines, the first control plane machine is created but subsequent machines are not created.
+
+### CPU requirements are not listed in the documentation (COPS-7168)
+
+This issue is corrected in the documentation.
+
+### machine stuck in provisioned state after pivot (COPS-7166)
+
+After performing a Pivot operation to migrate cluster resources in DKP 2.1.1, some machines could be stuck in the provisioning state.
+
+### kube-prometheus-stack error with fresh on-prem install (COPS-7163)
+
+[This fix](https://github.com/prometheus-operator/prometheus-operator/pull/4221) from Prometheus resolves the issue.
```

Comment:
+1 removing this section. IIRC the problem was that kube-prometheus-stack was trying to upgrade its CRDs to a new version with an invalid spec. We've since upgraded kube-prometheus-stack to pull in a fix for that, so this problem should no longer be encountered.

Reply:
I take it back, we fixed this in Kommander 2.2 but didn't backport to 2.1, I think because it was a big upgrade. So we probably need to keep this in 2.1 release notes.

Reply:
Excellent, thank you for double-checking @branden and @cneth ! Keeping in RN.

Reply:
```suggestion
### kube-prometheus-stack error with fresh on-prem install (COPS-7163)

A problem with Prometheus where the component `kube-prometheus-stack` attempted to upgrade its CRDs to a new version with invalid specifications [was fixed](https://github.com/prometheus-operator/prometheus-operator/pull/4221).
```

Reply:
How about this instead:

```suggestion
### kube-prometheus-stack error during error install (COPS-7163)

A problem with Prometheus where the component `kube-prometheus-stack` attempted to upgrade its CRDs to a new version with invalid specifications was solved.
```